### PR TITLE
Update builds controller to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**
@spadgett

### CODE REVIEW

File: app/scripts/controllers/builds.js [hunk 1]
```diff
@@ -8,7 +8,17 @@
...
+    $routeParams,
+    $scope,
+    APIService,
+    DataService,
+    $filter,
+    LabelFilter,
+    Logger,
+    $location,
+    BuildsService,
+    ProjectsService) {
```
Comment (spadgett) [inline app/scripts/controllers/builds.js]: Let's sort the dependencies to keep the lists manageable

Comment (benjaminapetersen) [reply app/scripts/controllers/builds.js]: sort done

**Comment by benjaminapetersen** [comment]
Yeah, this one didn't need a view update as there are no `canI` checks in the matching view.

File: app/scripts/controllers/builds.js [hunk 2]
```diff
@@ -20,6 +30,9 @@ angular.module('openshiftConsole')
 
     var buildConfigForBuild = $filter('buildConfigForBuild');
 
+    $scope.buildsVersion = APIService.getPreferredVersion('builds');
+    $scope.buildConfigsVersion = APIService.getPreferredVersion('buildconfigs');
```
Comment (spadgett) [inline app/scripts/controllers/builds.js]: Let's not use `$scope` vars if these aren't used in the view

Comment (benjaminapetersen) [reply app/scripts/controllers/builds.js]: sounds good.  kept that for consistency with other views, but agree, no point in blowing up `$scope`.

**Comment by benjaminapetersen** [comment]
updated

**Comment by spadgett** [comment]
[merge]

**Pull request merged**